markers/mrk_evenlyInBlocks.m

```diff
@@ -47,7 +47,7 @@
   new_time= blk.ival(1,bb)+opt.OffsetStart:msec:blk.ival(2,bb)-opt.OffsetEnd;
   nMrk= length(new_time);
   mrk.time= cat(2, mrk.time, new_time);
-  mrk.event.blkno= cat(2, mrk.event.blkno, bb*ones(1,length(new_time)));
+  mrk.event.blkno= cat(1, mrk.event.blkno, bb*ones(1,length(new_time))'); % blkno has to be column vector for later functions
```

Comment:
should rather be: 
  bb*ones(length(new_time), 1)
than generating a row vector first and transpose it.